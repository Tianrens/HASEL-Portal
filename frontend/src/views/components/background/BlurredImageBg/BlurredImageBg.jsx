import React from 'react';

import { bgImg } from './BlurredImageBg.module.scss';

function BlurredImageBg({ children }) {
    return <div className={bgImg}>{children}</div>;
}

export default BlurredImageBg;
