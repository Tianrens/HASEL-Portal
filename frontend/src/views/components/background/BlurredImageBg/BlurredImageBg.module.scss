$bg-img: url('./UoAClockTower.jpg');

.bgImg {
    background-image: $bg-img;
    background-size: cover;
    background-position: center;
    height: 100%;
}
