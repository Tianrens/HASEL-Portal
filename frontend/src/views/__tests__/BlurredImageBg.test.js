import React from 'react';
import renderer from 'react-test-renderer';
import { StylesProvider } from '@material-ui/core';
import BlurredImageBg from '../components/background/BlurredImageBg/BlurredImageBg';

test('Blurred Image Background renders properly', () => {
    const snapshotComponent = renderer.create(
        <StylesProvider injectFirst>
            <BlurredImageBg />
        </StylesProvider>,
    );
    const tree = snapshotComponent.toJSON();
    expect(tree).toMatchSnapshot();
});
