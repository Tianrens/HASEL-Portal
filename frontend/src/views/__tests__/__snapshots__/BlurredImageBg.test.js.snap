// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blurred Image Background renders properly 1`] = `
<div
  className="bgImg"
/>
`;
